qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -37,41 +43,80 @@ def _get_shared_links_for_study(study):
     return ", ".join(shared)
 
 
-def _build_study_info(studytype, user=None):
+def _build_study_info(studytype, user, results=None):
     """builds list of namedtuples for study listings"""
-    if studytype == "private":
-        studylist = user.user_studies
+    if studytype not in {"standard", "shared"}:
+        raise IncompetentQiitaDeveloperError("Must use standard or shared!")
```

Comment:
I might be an `IncompetentQiitaDeveloper`; but I'll always raise this error when I use this function because I do not know the meaning of the study_type parameter (standard? shared?).